### PR TITLE
Updated run error using #183 and initial Cypress Dashboard Service Setup

### PR DESCRIPTION
**Pull request by sannleen**
## What is the Purpose?
Initial Cypress Dashboard Service Setup

## Mentions?
@andrewtpham @ninetteadhikari @michaelbukachi

### CODE REVIEW

File: tests/e2e/specs/general-app/app-access/login.js [hunk 1]
```diff
@@ -5,18 +5,53 @@
 describe('Valid and invalid login', () => {
   context('Valid login', () => {
     let data
+    const login = (username, paswword) => {
```
Comment (ninetteadhikari) [inline tests/e2e/specs/general-app/app-access/login.js:8]: minor change request:
revise password spelling from 'paswword' to 'password'

same for below line: 9

File: tests/e2e/specs/general-app/app-access/login.js [hunk 2]
```diff
@@ -5,18 +5,53 @@
...
-    it('Logs in successfully.', () => {
+    beforeEach(() => {
       cy.visit('/')
+    })
 
-      cy.login(data.admin, data.adminPassowrd)
-      cy.get('.brand').should('exist')
-      cy.get('h1').should('contain', 'Dots')
+    it('Logs in successfully using username.', () => {
+      login(data.admin, data.adminPassowrd)
```
Comment (ninetteadhikari) [inline tests/e2e/specs/general-app/app-access/login.js:34]: same revise from 'adminPassowrd' to 'adminPassword'

and same for line 38, 42, 46, 50, 54

Comment (sannleen) [reply tests/e2e/specs/general-app/app-access/login.js]: Typos fixed

File: tests/e2e/specs/general-app/app-access/login.js (hunk 1 above)
Comment (sannleen) [reply tests/e2e/specs/general-app/app-access/login.js]: fixed

**Pull request merged**